react/features/shared-iframe/components/SharedIFrame.js

```diff
@@ -0,0 +1,181 @@
+// @flow
+
+import React from 'react';
+
+import Filmstrip from '../../../../modules/UI/videolayout/Filmstrip';
+import { i18next, DEFAULT_LANGUAGE } from '../../base/i18n';
+import { getLocalParticipant } from '../../base/participants';
+import { connect } from '../../base/redux';
+import { getToolboxHeight } from '../../toolbox/functions.web';
+import { getGenericiFrameUrl } from '../functions';
+
+import IFrameManager from './IFrameManager';
+
+
+declare var interfaceConfig: Object;
+
+type Props = {
+
+    /**
+     * The available client width.
+     */
+    clientHeight: number,
+
+    /**
+     * The available client width.
+     */
+    clientWidth: number,
+
+    /**
+     * Whether the (vertical) filmstrip is visible or not.
+     */
+    filmstripVisible: boolean,
+
+    /**
+     * The id of the local participant.
+     */
+     _localParticipantId: string,
+
+    /**
+     * The current shared iframe state.
+     *
+     * @private
+     */
+     _sharedIFrames: Object,
+
+    /**
+     * The current users room.
+     *
+     * @private
+     */
+    _room: string,
+
+     /**
+     * The current users language setting.
+     *
+     * @private
+     */
+     _lang: string,
+
+    /**
+     * The templateUrl of the IFrame which should be displayed.
+     */
+     shareUrl: string,
+}
+
+/** .
+ * Implements a React {@link Component} which represents the large video (a.k.a.
+ * The conference participant who is on the local stage) on Web/React.
+ *
+ * @augments Component
+ */
+class SharedIFrame extends React.PureComponent<Props> {
+    /**
+     * Computes the width and the height of the component.
+     *
+     * @returns {{
+     *  height: number,
+     *  width: number
+     * }}
+     */
+    getDimensions() {
+        const { clientHeight, clientWidth, filmstripVisible } = this.props;
+
+        let width;
+        let height;
+
+        if (interfaceConfig.VERTICAL_FILMSTRIP) {
```

Comment:
This breaks mobile.

Reply:
what exactly is breaking? 
I have no problems opening it in chrome emulated as pixel 5

Reply:
![image](https://user-images.githubusercontent.com/13590797/166216462-c1085924-0a12-482f-a75a-7c14312d6484.png)

Reply:
The mobile app, not the mobile web view.